src/renderers/dom/shared/devtools/ReactDOMSVGDeprecatedAttributeDevtool.js

```diff
@@ -33,6 +33,7 @@ if (__DEV__) {
         warnedSVGAttributes.hasOwnProperty(name) && warnedSVGAttributes[name]) {
       return;
     }
+    warnedSVGAttributes[name] = true;
```

Comment:
Not a deal breaker, but seems a bit weird that `warnedSVGAttributes` gets set to true even if a warning never fired (ie. bailed out in the next three lines).  Maybe a few of these lines can be re-arranged to avoid this, maybe not worth the headache.